tests/trainer/logging/test_logger_connector.py

```diff
@@ -178,18 +182,20 @@ def train_dataloader(self):
     )
     trainer.fit(model)
 
-    assert len(trainer.logger_connector.cached_results("train")['training_step']['0']['0']['0']) == len(model.train_losses)
+    train_results = model.train_results
+
+    assert len(train_results['training_step']['0']['0']['0']) == len(model.train_losses)
```

Comment:
as discussed with @SeanNaren and @tchaton lets find a simpler way to access this content e.g.
`train_results(hook='training_step', dataloader='0', batch='0', ...)`

Reply:
Done !